File: higher-order-binary-ops/less-than-100.dart
main() {
  var lt  = (ro) => (lo) => lo < ro;
  var num = [5, 500, 50, 101, 200, 35];
  var out = num.where(lt(100)).toList();

  print('less than 100: ${out}');
  //=> less than 100: [ 5, 50, 35 ]
}

File: higher-order-binary-ops/less-than-100.php
<?php
function lt ($ro) {
  return function ($lo) use($ro) { return $lo < $ro; };
}

$num = [5, 500, 50, 101, 200, 35];
$out = array_filter($num, lt(100));

printf('less than 100: [ %s ]', join(', ', $out));
//=> less than 100: [ 5, 50, 35 ]
